Performance/.gitignore

```diff
@@ -0,0 +1,2 @@
+Generated/
+Results/
```

Comment:
Thanks for segregating built content into separate directories like this.

Recently, I've started naming directories like this with a leading underscore, e.g. `_generated` or `_results`.  That helps distinguish them from directories that are part of the repo.